frontend/src/features/SignIn/signInAPI.ts

```diff
@@ -0,0 +1,7 @@
+import axios from "../../axios/api";
+
+export const signInUser = (data: { email: string; password: string }) => {
+  return axios.post("/api/users/signin", data).catch(function (error: any) {
```

Comment:
you can try this to put the datatype of the error instead of using any.
```js
.catch((err: Error | AxiosError) {
  if (axios.isAxiosError(error))  {
    // Access to config, request, and response
  } else {
    // Just a stock error
  }
})
```